cpp/src/io/csv/csv_reader.cu

```diff
@@ -960,14 +948,15 @@ gdf_error launch_dataConvertColumns(raw_csv_t *raw_csv, void **gdf, gdf_valid_ty
 	int gridSize = (raw_csv->num_records + blockSize - 1) / blockSize;
 
 	parsing_opts_t opts;
-	opts.delimiter		= raw_csv->delimiter;
-	opts.terminator		= raw_csv->terminator;
-	opts.quotechar		= raw_csv->quotechar;
-	opts.keepquotes		= raw_csv->keepquotes;
-	opts.decimal		= raw_csv->decimal;
-	opts.thousands		= raw_csv->thousands;
-
-	convertCsvToGdf <<< gridSize, blockSize >>>(
+	opts.delimiter			= raw_csv->delimiter;
```

Comment:
If `parsing_opts_t` were a C++ struct, there could be a `parsing_opts_t` constructor that accepts a `raw_csv_t` that does this initialization.

Reply:
I've changed the usage of this struct so we only use it once now in read_csv(). I can still make a constructor, but I didnt't want to move the assignment code into a different file for now.